src/components/Tones/SoundButton.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { Sampler } from 'tone';
+
+const SoundButton = props => {
+	const [isLoaded, setLoaded] = useState(false);
+	const sampler = useRef(null);
+	const A1 = props.sound;
+
+	useEffect(() => {
+		sampler.current = new Sampler(
+			{ A1 },
+			{
+				onload: () => {
+					setLoaded(true);
+				},
+			}
```

Comment:
Don't understand how this part works. 
Also just testing the comments :)
Looks good